draft-ietf-rtcweb-jsep.xml

```diff
@@ -2765,8 +2796,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                   "sendonly", prepare to start transmitting media using the specified primary SSRC
                   and one of the selected payload types, once the underlying transport
                   layers have been established. Otherwise, stop transmitting RTP media,
-                  although RTCP should still be sent, as described in
-                  <xref target="RFC3264"/>, Section 5.1. </t>
+                  although RTCP should still be sent, as described in 
+                  <xref target="RFC3264"/>, Section 5.1. If RID values are specified, include the RID header
+		  extension in the RTP streams, as indicated in <xref
+		  target="I-D.ietf-mmusic-rid"></xref>, Section 4).  If simulcast is negotiated,
+		  send the number of Source RTP Streams as specified in
+		  <xref target="I-D.ietf-mmusic-sdp-simulcast"></xref>, Section 6.2.2.
+		  The "send" portion of any "a=simulcast" line from a
+		  remote description MUST be ignored, since receiving
+		  of simulcast is not supported.
```

Comment:
This is not a comment you need to deal with at all but just as a random idea ... I wonder if we really need to forbid receiving simulcast in a browser. The objects would now allow that and if you want to to do a conference where one of the browsers was elected as a MCU style conference bridge and send media to and from the other browsers, .... well that might be cool. Not a use case I care about but we might want to leave the door open to something like this.